HW_1/task_1.py

```diff
@@ -0,0 +1,16 @@
+# Задача 2: Найдите сумму цифр трехзначного числа
+
+n = int(input("Введите трехзначное число n: "))
+
+if (99 < n < 1000):
+    a = n % 10
+    b = round((n-a)/10 % 10)
+    c = round((n - a - b*10)/100)
```

Comment:
И тут, соответственно, тоже.
```suggestion
    c = n // 100)
```